app/controllers/api/v1/user_events_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::UserEventsController < ApplicationController
+  protect_from_forgery with: :null_session
+  
+  def create
```

Comment:
This works great right now! Maybe we can look into refactoring this moving forward.